app-conf/FetcherConf.xml

```diff
@@ -36,6 +36,21 @@
       <sampling_enabled>false</sampling_enabled>
     </params>
   </fetcher>
+  <!--
+    This is an experimental replacement for the MapReduceFetcherHadoop2 that attempts to burn
+    through queues of jobs faster by pulling data directly from HDFS rather than going through the
+    job history server.
+  -->
+  <!--
+  <fetcher>
+    <applicationtype>mapreduce</applicationtype>
+    <classname>com.linkedin.drelephant.mapreduce.MapReduceFSFetcherHadoop2</classname>
+    <params>
+      <sampling_enabled>false</sampling_enabled>
+      <history_log_size_limit_in_mb>100</history_log_size_limit_in_mb>
```

Comment:
Can you add a comment saying the impact of increasing the log size limit. And also, if people reach OOM then how can they increase the JVM heap size to analyze jobs having large logs.